### ISSUE TITLE
Parsing response issues with 1.4.0

### ISSUE THREAD

**Issue by cbakker** [helpscout/helpscout-api-php#17]
Started using this client an hour ago and found some bugs. Now I found out that 1.4.0 is not released yet and is probably a development version. But maybe I am the only one getting these errors, so I thought I will submit them here:

**Undefined offset: 1 in vendor/helpscout/api/src/HelpScout/ApiClient.php on line: 743**

When I dumped the headers, it started with this:
```
HTTP/1.1 100 Continue

HTTP/1.1 201 Created
Date: Fri, 03 Apr 2015 17:00:07 GMT
Server: Apache
Access-Control-Allow-Origin: *
Location: https://api.helpscout.net/v1/conversations/81977864.json
```

So this returned 2 HTTP headers, maybe it was redirected or something. But this made rule 743 fail. I added this around rules 743 and 744 and it worked:

```
if (strstr($line, ': ')) {
```

Now I get this error: **substr() expects parameter 1 to be string, array given vendor/helpscout/api/src/HelpScout/ApiClient.php on line: 899**

And then I found out this was a very recent version. So I will now try the 1.3 version.

**Comment by cbakker** [comment]
I can now confirm that with the exact same code, but with 1.3.11 it works perfectly fine.

**Comment by stevenwadejr** [comment]
@cbakker Thanks for pointing this out. Our testing missed that. The 1.4.0 release has an extremely short life. Version 1.5.0 is refactoring and rewriting the cURL interactions and will address this bug. It'll be released in a short time. I'll ping you back here when it's live.

**Comment by cbakker** [comment]
Looking forward to that! :)

**Comment by stevenwadejr** [comment]
Hi @cbakker,

Thanks for the PR! We've merged it in and have released 1.5.0 which should address this issue. Let me know if you're still having problems.

**Comment by cbakker** [comment]
1.5.0 now works perfectly here.
